environment/config.js

```diff
@@ -40,7 +39,8 @@ requirejs.config({
         'jquery.mockjax': '/node_modules/jquery-mockjax/dist/jquery.mockjax',
 
         'lib/decimal/decimal': '/node_modules/decimal.js/decimal',
-        'lib/expr-eval/expr-eval': '/node_modules/@oat-sa/expr-eval/dist/bundle'
+        'lib/expr-eval/expr-eval': '/node_modules/@oat-sa/expr-eval/dist/bundle',
+        async: '/node_modules/async/lib/async'
```

Comment:
Could you please keep `async` inside quotes ? Just because it's now a reserved keyword `'async'` and this is a js file